runtime/manual/tools/publish.md

```diff
@@ -0,0 +1,131 @@
+# deno publish
+
+*This applies to `deno` v.1.17.0. and above.* **Unstable preview feature**.
```

Comment:
```suggestion
*This applies to `deno` v1.42.0. and above.* **Unstable preview feature**.
```

Reply:
I don't think we should mark it as "unstable preview"